### ISSUE TITLE
Unable to set socket option: Permission denied

### ISSUE THREAD

**Issue by CodeMyst** [shove70/dbc#2]
```
std.socket.SocketOSException@std/socket.d(3185): Unable to set socket option: Permission denied
----------------
??:? @trusted void std.socket.Socket.setOption(std.socket.SocketOptionLevel, std.socket.SocketOption, void[]) [0xd476dfb7]
??:? @trusted void std.socket.Socket.setOption(std.socket.SocketOptionLevel, std.socket.SocketOption, int) [0xd476dff8]
/home/codemyst/.dub/packages/dbc-1.1.7/dbc/src/mysql/socket.d:18 void mysql.socket.Socket.connect(const(char)[], ushort) [0xd4529b20]
/home/codemyst/.dub/packages/dbc-1.1.7/dbc/src/mysql/connection.d:607 void mysql.connection.Connection.connect() [0xd45241c0]
/home/codemyst/.dub/packages/dbc-1.1.7/dbc/src/mysql/connection.d:119 mysql.connection.Connection mysql.connection.Connection.__ctor(immutable(char)[], mysql.protocol.CapabilityFlags) [0xd452356b]
/home/codemyst/.dub/packages/dbc-1.1.7/dbc/src/mysql/connection.d:111 mysql.connection.Connection mysql.connection.Connection.__ctor(immutable(char)[]) [0xd45234fc]
source/app.d:263 _Dmain [0xd448a53a]
Program exited with code 1
```

But it works when run with sudo. Any clues to why this might be happening?

**Comment by CodeMyst** [comment]
This was fixed in #1 but a release needs to be pushed to dub.

**Comment by shove70** [comment]
Sorry, it was not handled in time.
